### PR TITLE
Add narze keymaps for Ergodox & Planck with SuperDuper mode implementation

### PR DESCRIPTION
**Pull request by narze**

### CODE REVIEW

File: keyboards/planck/keymaps/narze/Makefile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
+
+COMBO_ENABLE = yes
+
+ifndef QUANTUM_DIR
+	include ../../../../Makefile
+endif
```
Comment (jackhumbert) [inline keyboards/planck/keymaps/narze/Makefile]: This file isn't needed anymore, and is ignore by the build system :)

File: keyboards/ergodox/keymaps/narze/Makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
+
+# Combos not working yet
+COMBO_ENABLE = no
+
+ifndef QUANTUM_DIR
+	include ../../../../Makefile
+endif
```
Comment (jackhumbert) [inline keyboards/ergodox/keymaps/narze/Makefile]: Same with this one!

**Comment by narze** [comment]
@jackhumbert I've pushed the fix, thanks

**Comment by jackhumbert** [comment]
Sorry, I meant the whole `Makefile` for both comments - I should have been clearer! :)

**Comment by narze** [comment]
Oh, and where can I put the vars instead?

**Comment by drashna** [comment]
All the makefile stuff should be in a "rules.mk" file. In fact, you should be able to just rename the makefile to rules.mk and be set.

**Comment by narze** [comment]
🙆‍♂️

**Comment by jackhumbert** [comment]
Cool! Thanks :)

**Pull request merged**